src/client/containers/Home/Home.styles.css

```diff
@@ -204,11 +225,31 @@
 }
 
 @media (max-width: 1020px) {
+  header.homepage-header h2 {
+    font-size: 1.5rem !important;
```

Comment:
Avoid using !important and rather have specific css, do you need` !important `here?

Reply:
We will go with the `!important` this time ;)